webvtt/parsing/file-parsing/support/stylesheets-newlines.test

```diff
@@ -0,0 +1,46 @@
+stylesheets, rules
+<link rel="help" href="https://w3c.github.io/webvtt/#collect-a-webvtt-block">
+
+assert_equals(document.styleSheets.length, 1);
```

Comment:
That kinda sucks... I'll rename stylesheets-newline -> stylesheets and have that as the only parsing test for stylesheet.